website/docs/registry/modules/verified.mdx

```diff
@@ -1,25 +1,25 @@
 ---
-page_title: Terraform Registry - Verified Modules
+page_title: Terraform Registry - Partner Modules
 description: >-
-  Verified modules are reviewed by HashiCorp and actively maintained by
-  contributors to stay up to date.
+  Partner modules are actively maintained by partner
+  contributors.
 ---
 
-# Verified Modules
+# Partner Modules
 
-Verified modules are reviewed by HashiCorp and actively maintained by contributors to stay up-to-date and compatible with both Terraform and their respective providers.
+Partner modules are published by tech and cloud organizations that have a partnership agreement with HashiCorp. Partner contributors actively maintain their modules to stay up-to-date and compatible with both Terraform and their respective providers.
```

Comment:
Suggesting a rewrite of the first sentence to be in active voice: 
```suggestion
Organizations that have a partnership agreement with HashiCorp can publish partner modules. Partner contributors actively maintain their modules to stay up-to-date and compatible with both Terraform and their respective providers.
```

Reply:
Do we want to mention or link to the [Partner program](https://www.hashicorp.com/partners/become-a-partner?ajs_aid=b4683bed-199f-451d-ad25-9546dbd0ec8f&product_intent=terraform) here? No worries if not, I'm just curious.